cluster-manifests/cluster-rbac.yaml

```diff
@@ -1,24 +1,27 @@
-#  Mapping k8s cluster user facing roles to AAD groups: https://kubernetes.io/docs/reference/access-authn-authz/rbac/#user-facing-roles
-#apiVersion: rbac.authorization.k8s.io/v1
-#kind: ClusterRoleBinding
-#metadata:
-#  name: cluster-admins
-#roleRef:
-#  apiGroup: rbac.authorization.k8s.io
-#  kind: ClusterRole
-#  name: cluster-admin
-#subjects:
-#  - kind: Group
-#    name: <replace-with-an-aad-group-object-id-for-this-cluster-role-binding>
-#---
-#apiVersion: rbac.authorization.k8s.io/v1
-#kind: ClusterRoleBinding
-#metadata:
-#  name: cluster-viewers
-#roleRef:
-#  apiGroup: rbac.authorization.k8s.io
-#  kind: ClusterRole
-#  name: view
-#subjects:
-#  - kind: Group
-#    name: <replace-with-an-aad-group-object-id-for-this-cluster-role-binding>
+# This file is exclusively to be used when you are using Azure AD direct assignment for Kubertnetes RBAC 
+# and NOT when you are using Azure RBAC as your Kubernetes RBAC backing store.
+#
+#  Mapping k8s cluster user facing roles to AAD groups: https://kubernetes.io/docs/reference/access-authn-authz/rbac/#user-facing-roles
+#apiVersion: rbac.authorization.k8s.io/v1
+#kind: ClusterRoleBinding
+#metadata:
+#  name: cluster-admins
+#roleRef:
+#  apiGroup: rbac.authorization.k8s.io
+#  kind: ClusterRole
+#  name: cluster-admin
+#subjects:
+#  - kind: Group
+#    name: <replace-with-an-aad-group-object-id-for-this-cluster-role-binding>
+#---
+#apiVersion: rbac.authorization.k8s.io/v1
+#kind: ClusterRoleBinding
+#metadata:
+#  name: cluster-viewers
+#roleRef:
+#  apiGroup: rbac.authorization.k8s.io
+#  kind: ClusterRole
+#  name: view
+#subjects:
+#  - kind: Group
+#    name: <replace-with-an-aad-group-object-id-for-this-cluster-role-binding>

```

Comment:
Any reason we have all the line ending changes on this file?  (Or is the GitHub UI showing the whole file changed for another reason?)  For example, the a0008 namespace file only had the top message added, as expected.

Reply:
Not sure what the problem is. I removed the extra line at the end of the file, but the UI is still showing the entire file changed.